srcpkgs/Aegisub/template

```diff
@@ -1,52 +1,51 @@
 # Template file for 'Aegisub'
 pkgname=Aegisub
-version=3.2.2
-revision=12
-build_style=gnu-configure
-configure_args="--disable-update-checker --with-alsa --with-ffms2
- --with-wx-config=wx-config-gtk3 $(vopt_with fftw fftw3) $(vopt_with openal)
- $(vopt_with portaudio)"
-hostmakedepends="automake gettext-devel intltool libtool LuaJIT pkg-config"
-makedepends="alsa-lib-devel boost-devel libcurl-devel fontconfig-devel
- freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel LuaJIT-devel
- MesaLib-devel wxWidgets-gtk3-devel $(vopt_if fftw fftw-devel)
- $(vopt_if openal libopenal-devel) $(vopt_if portaudio portaudio-devel)"
-short_desc="Tool for creating and modifying subtitles"
-maintainer="Enno Boland <gottox@voidlinux.org>"
+version=3.3.2
+revision=1
+_githash=723d01d13088d9b1bccc821f1129d045a0b74e2c
+wrksrc="${pkgname}-${version}"
+build_style=cmake
+cmake_builddir="BUILD"
+configure_args="
+	-DwxWidgets_CONFIG_EXECUTABLE=/usr/bin/wx-config-gtk3
+	-DWITH_PORTAUDIO=$(vopt_if portaudio ON OFF)
+	-DWITH_OPENAL=$(vopt_if openal ON OFF)
+	-DWITH_FFTW3=$(vopt_if fftw ON OFF)
+"
+hostmakedepends="automake gettext-devel intltool libtool LuaJIT pkg-config git"
+makedepends="
+	alsa-lib-devel
+	boost-devel
+	libcurl-devel
+	fontconfig-devel
+	freetype-devel
+	hunspell-devel
+	icu-devel
+	libass-devel
+	libffms2-devel
+	LuaJIT-devel
+	MesaLib-devel
+	wxWidgets-gtk3-devel
+	$(vopt_if fftw fftw-devel)
+	$(vopt_if openal libopenal-devel)
+	$(vopt_if portaudio portaudio-devel)
+"
+short_desc="Subtitle editor (ssa, ass, srt) (wangqr fork)"
+maintainer="pudiva <rogi@skylittlesystem.org>"
 license="BSD-3-Clause, MIT"
-homepage="http://www.aegisub.org"
-distfiles="https://github.com/Aegisub/Aegisub/archive/v${version}.tar.gz"
-checksum=62757dd491455268a240f983b59734a801cc2e899039a7493deeaf5e24a61dcd
+homepage="https://github.com/wangqr/Aegisub"
+conflicts=Aegisub
```

Comment:
this is an old artifact from when this was a separate package, now Aegisub shouldn't conflict with Aegisub